composer.json

```diff
@@ -45,10 +45,13 @@
         "apigen": [
             "php devtools/vendor/bin/apigen -c build/config/apigen.neon -d build/api -s src -s vendor/symfony/config -s vendor/symfony/console -s vendor/symfony/yaml -s vendor/guzzle/guzzle/src -s vendor/psr/log --report build/logs/checkstyle-apigen.xml --exclude *Test.php --exclude */Tests/* --google-analytics UA-40398434-1"
         ],
+        "install-box": [
+            "curl -LSs http://box-project.github.io/box2/installer.php | php"
+        ],
         "box": [
             "mkdir -p build/artifacts",
```

Comment:
agh. Windows users can do similar by themselves.

I'm fine with this, but if it is a problem for you, I can do either

- Stop using composer script. Just use .travis.yml
- Write a portable mkdir wrapper and use it in composer.json